src/stream.ts

```diff
@@ -214,10 +213,10 @@ export class DataAndMoneyStream extends Duplex {
     }
     const sendMax = longFromValue(limit, true)
     if (this._totalSent.greaterThan(sendMax)) {
-      this.log.debug(`cannot set sendMax to ${sendMax} because we have already sent: ${this._totalSent}`)
+      this.log.debug('cannot set sendMax to %s because we have already sent: %s', sendMax, this._totalSent)
```

Comment:
Is there a performance difference between the two syntax's?  (just personal interest)

Reply:
Yes, because it isn't just different syntax's. 

Log formatters (e.g. `%s`, `%d`, …) only stringify their arguments when the corresponding log level is actually enabled. Since debug and trace are usually not needed, this can save quite a bit of work.

String interpolation (via template literals in this case) serializes the variables even when logging is disabled.

Reply:
@matdehaast 
Especially the custom `%h` @sentientwaffle added for formatting Buffers